deploy/helm/kuberhealthy/Chart.yaml

```diff
@@ -1,7 +1,7 @@
 apiVersion: v2
 name: kuberhealthy
-appVersion: v2.7.1
-version: 83
+appVersion: v2.7.2
```

Comment:
Don't rev this version - it will cause the wrong kuberhealthy image to be pulled...

Reply:
Guess you mean don't increment/revise this. Let me keep it to v2.7.1 only.

Reply:
Updated back to v2.7.1